File: test/pc/Lamp/LampEvents.cs
/*
* Project Title : inHausOSGiGateway
* Author        : Lohrasb jalali
* Company       : Fraunhofer IMS
*/
namespace de.inhaus.inHausLamp
{
	using System;
	
	/// <summary> 
	/// 
	/// </summary>
	/// <author>  Lohrasb jalali
	/// </author>
	/// <version>  
	/// @since 
	/// 
	/// </version>
	public interface LampEvents
		{
			void  stateChanged(System.String deviceID, bool state);
			void  intensityChanged(System.String deviceID, int intensity);
		}
}

File: test/pc/Lamp/inHausLamp(java)/LampEvents.java
/*
 * Project Title : inHausOSGiGateway
 * Author        : Lohrasb jalali
 * Company       : Fraunhofer IMS
 */

package de.inhaus.inHausLamp;

/**
 * 
 * 
 * @author Lohrasb jalali
 * @version 
 * @since 
 */
public interface LampEvents {
  
  public void stateChanged(String deviceID,boolean state);
  public void intensityChanged(String deviceID,int intensity);

}
